src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -32,15 +32,170 @@
+ *         start_column = 2
+ *
+ *         # Required field for locatableXSV files.
+ *         # The 0-based index of the column containing the end position for each row
+ *         end_column = 3
+ *     </pre>
+ * </p>
+ *
+ * <h3>Inputs</h3>
+ * <ul>
+ *     <li>A reference genome sequence.</li>
+ *     <li>The version of the reference genome sequence being used (either <i>hg19</i> or <i>hg38</i>).</li>
+ *     <li>A VCF of variant calls to annotate.</li>
+ *     <li>The path to a folder of data sources formatted for use by Funcotator.</li>
+ * </ul>
+ *
+ * <h3>Output</h3>
+ * <ul>
+ *     <li>A VCF containing all variants from the input file with added annotation columns corresponding to annotations from each data source that matched a given variant according to that data source's matching criteria.</li>
+ * </ul>
+ *
+ * <h3>Usage example</h3>
+ * <pre>
+ *   ./gatk Funcotator \
+ *   -R reference.fasta \
+ *   -V input.vcf \
+ *   -O output.vcf \
+ *   --data-sources-path dataSourcesFolder/ \
+ *   --ref-version hg19
+ * </pre>
+ *
+ * <h3>Notes</h3>
+ * <ul>
+ *     <li>This is a beta tool, and as such may generate errors or warnings.</li>
+ *     <li>This tool is the GATK analog of <a href="http://portals.broadinstitute.org/oncotator/">Oncotator</a>.</li>
+ * </ul>
+ *
+ * <h3>Known Issues</h3>
+ * <p>A complete list of known open issues can be found on <a href="https://github.com/broadinstitute/gatk/issues?q=is%3Aopen+is%3Aissue+label%3AFuncotator">the GATK github entry for funcotator here.</a></p>
+ *
+ * <h4>Notable Issues as of 2018 Jan 3</h4>
+ * <ul>
+ *     <li>Only supports VCF for inputs and outputs (<a href="https://github.com/broadinstitute/gatk/issues/3922">Issue 3922</a>).</li>
+ *     <li>Only supports a single GENCODE data source (<a href="https://github.com/broadinstitute/gatk/issues/3956">Issue 3956</a>).</li>
+ *     <li>Non-GENCODE annotations on multiallelic variants are only rendered properly for the last allele (<a href="https://github.com/broadinstitute/gatk/issues/3896">Issue 3896</a>).</li>
+ *     <li>The "other transcripts" annotation is missing for IGR variants (<a href="https://github.com/broadinstitute/gatk/issues/3849">Issue 3849</a>).</li>
+ *     <li>Only the "Format" field of input VCF files is preserved in the output VCF file (<a href="https://github.com/broadinstitute/gatk/issues/3895">Issue 3895</a>).</li>
+ *     <li>Codon Change and Protein Change for indels spanning splice sites are not properly rendered (<a href="https://github.com/broadinstitute/gatk/issues/3749">Issue 3749</a>).</li>
+ * </ul>
  *
- * Created by jonn on 8/22/17.
  */
 @CommandLineProgramProperties(
-        summary = "Create functional annotations on given variants cross-referenced by a given database.\n" +
+        summary = "Create functional annotations on given variants cross-referenced by a given set of data sources.\n" +
                 "A GATK version of the Oncotator.",
```

Comment:
First of all, it's just "Oncotator" not "the Oncotator".  But I am not sure you want to keep this line.  Or maybe write "similar functionality to Oncotator".

Reply:
Sounds good.  Fixed!

Reply:
Sounds good.  Fixed!